src/Cache.zig

```diff
@@ -16,7 +16,8 @@ const Allocator = std.mem.Allocator;
 /// This protection is conditionally compiled depending on `want_debug_deadlock`.
 var all_cache_digest_set: std.AutoHashMapUnmanaged(BinDigest, void) = .{};
 var all_cache_digest_lock: std.Mutex = .{};
-const want_debug_deadlock = std.debug.runtime_safety;
+// TODO: Figure out how to make sure that `all_cache_digest_set` does not leak memory!
+pub const want_debug_deadlock = false;
```

Comment:
@andrewrk To make sure that the compiler doesnt leak memory we have to call `deinit` on `all_cache_digest_set`, however when I tried to do that I ran into segfaults. I'm not sure how to solve that.

Reply:
The call should go into cleanExit in main.zig. If you show how to repro the segfault I'll take a look